misc/assets/anonymization.svg

```diff
@@ -0,0 +1,62 @@
+<svg version="1.1" viewBox="0 0 1000 400" xmlns="http://www.w3.org/2000/svg">
+  <style>
+      text {
+        font-family: sans-serif;
+      }
+      .caption {
+        font-size: 12pt;
+        text-anchor: middle;
+        dominant-baseline: central;
+      }
+      .relation {
+        fill: none;
+        stroke: black;
+        stroke-width: 1pt;
+      }
+      .relation.from-many {
+        marker-start: url(#relation-many)
+      }
+  </style>
+
+  <defs>
+    <marker id="relation-many" orient="auto" markerWidth="6" markerHeight="6" refX="0" refY="3">
+      <circle cx="3" cy="3" r="3" />
+    </marker>
+    <g id="model-wide">
+      <rect class="box" x="-125" y="-20" width="250" height="40" fill="none" stroke="black" stroke-width="1pt" />
+    </g>
+    <g id="external">
+      <rect class="box" x="-100" y="-20" width="200" height="40" fill="none" stroke="black" stroke-width="1pt" stroke-dasharray="3pt" />
+    </g>
+  </defs>
+
+  <g transform="translate(300 100)">
+    <use href="#model-wide" />
+    <text class="caption">AttachmentNormalization</text>
+    <path class="relation from-many to-one" d="M 125,0 300,0" />
+  </g>
+  <g transform="translate(700 100)">
+    <use href="#external" class="wide" />
+    <text class="caption">Attachment</text>
+  </g>
+  
+  <g transform="translate(300 200)">
+    <use href="#model-wide" />
+    <text class="caption">AttachmentRecognition</text>
+    <path class="relation from-many to-one" d="M 125,0 300,0" />
+  </g>
+  <g transform="translate(700 200)">
+    <use href="#external" class="wide" />
+    <text class="caption">Attachment</text>
```

Comment:
Pls vsetky tri kopie modelu `Attachment` zjednot do jednej kopie. A vetky tri relacie ved to tej jednej kopie. Ciary relacii rob lomene kolme. Idealne tak, ze v strede medzi modelmi sa zalomia vertikalne, stretnu sa na jednom mieste a potom idu podnad seba. Ako keby sa tri ciary spajali do jednej.

V povodnom odt som na to zabudol. In general v kazdej scheme by sa kazdy model mal nachadzat naviac raz. Nech zo schemy vidno vzajomne vztahy modelov, co su s nim v realcii.